source/whats-new.txt

```diff
@@ -51,12 +51,16 @@ Learn what's new in:
 What's New in 2.0
 -----------------
 
-The 2.0 {+driver-short+} release supports various new features, provides
-a smoother timeout experience, and implements clearer, more idiomatic Go
+The 2.0 {+driver-short+} release supports OpenID Connect (OIDC)
+authentication, provides a smoother Client-Side Operations Timeout
```

Comment:
S: we don't capitalize this elsewhere, though I'm not sure which is correct. See, e.g., https://www.mongodb.com/docs/languages/python/pymongo-driver/current/connect/csot/

```suggestion
authentication, provides a smoother client-side operations timeout
```